app/src/main/java/com/github/polybooks/core/Books.kt

```diff
@@ -29,7 +29,9 @@ data class Book(
     val language : String?,
     val publisher : String?,
     val publishDate : Timestamp?,
-    val format : String?
+    val format : String?,
+    val totalStars: Double? = null,
```

Comment:
I would not put the rating with the Book. It makes it no longer immutable, et complicates the cache a lot.